pkg/middleware/middleware.go

```diff
@@ -21,8 +21,9 @@ var (
 	ReqOrgAdmin            = RoleAuth(org.RoleAdmin)
 )
 
-func HandleNoCacheHeader(ctx *contextmodel.ReqContext) {
-	ctx.SkipCache = ctx.Req.Header.Get("X-Grafana-NoCache") == "true"
+func HandleNoCacheHeaders(ctx *contextmodel.ReqContext) {
+	ctx.SkipDSCache = ctx.Req.Header.Get("X-Grafana-NoCache") == "true"
+	ctx.SkipQueryCache = ctx.Req.Header.Get("X-Cache-Skip") == "true"
```

Comment:
Where's this used, enterprise?

Reply:
Yes - it seemed closely related enough that I decided to put it here, though I could just read this header from the requests each time in the enterprise impl. 

Do you think it would be sufficient to just make a comment here, or do you think I should remove this from OSS?

Reply:
`X-Grafana-NoCache` and `X-Cache-Skip` looks very similar, `SkipDSCache` and `SkipQueryCache` make things more understandable. Think it's fine to keep, but come comments could be good I think.

Reply:
Added comments: [b51bfb5](https://github.com/grafana/grafana/pull/65616/commits/b51bfb54214a4c53c8f1cddaa84d35b9f83b7acd)